app/serializers/NPQ/participant_profile_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module NPQ
+  class ParticipantProfileSerializer
+    include JSONAPI::Serializer
+    include JSONAPI::Serializer::Instrumentation
+
+    set_type "npq-participant-profile"
+
+    attribute(:participant_id) do |npq_participant_profile|
+      npq_participant_profile.npq_application.user_id
+    end
+
+    attribute(:course_identifier) do |npq_participant_profile|
+      npq_participant_profile.npq_course.identifier
+    end
+
+    attributes(:training_status, &:state)
```

Comment:
I don't think `&:state` is necessary here - at least on the latest develop branch we store the current training status on the participant profile to avoid problems with ordering and nested queries in Rails.